test/list/cf_header.md

```diff
@@ -0,0 +1,14 @@
+# Kubernetes Conformance Test Suite - v1.7
+
+## Summary
+This document provides a summary of the tests included in the Kubernetes conformance test suite.
+Each test lists a set of formal requirements that a conformance platform must adhere to.
+
+The tests are a subset of the "e2e" tests that make up the Kubernetes testing infrastructure. 
+Each test is identified by the presence of the `[Conformance]` keyword in the ginkgo descriptive function calls.
+The contents of this document are extracted from comments preceding those `[Conformance]` keywords
```

Comment:
I would like to see a very simple example of what the comment preceding the [Conformance] tag would look like. Ideally a simple example makes it clear what these should look like and people will understand what we are adding and why.

Reply:
I have added an example into the header file.